docs/runbooks/CDINotReady.md

```diff
@@ -1,32 +1,41 @@
 # CDINotReady
+<!--Edited by davozeni, 17.10.2022-->
```

Comment:
Erm, I kinda forgot to do this part in my PRs, so thank you for reminding me :-) Generally speaking, though, it should be reasonably clear from PRs and Git history what has already been edited, but I guess a bit of redundancy can't hurt...

Reply:
I just added it, because Avital mentioned it in the guidelines she prepared for us :-)

Reply:
@jherrman The reason I requested this is so that I can search easily in a script for runbooks that have been edited. It's harder to automate if I need to use the Git history.